packages/sync/src/jvmMain/kotlin/io/realm/sync/transport/NetworkTransport.kt

```diff
@@ -0,0 +1,2 @@
+package io.realm.sync.transport
```

Comment:
Can't we have a common one for Android and JVM in `src/jvm/kotlin` instead? See the updated kmm-setup for `library` in master. 

Maybe there are some `ktor` dependencies that aren't resolvable if we do it that way, but then it might work by just adding `src/jvm/kotlin` to both `androidMain` and `jvmMain` source sets.

Reply:
I haven't added anything related to Ktor in this PR yet, it will come in the next one so I'll have a look at that then.